pkgs/games/rare/default.nix

```
@@ -13,13 +14,13 @@
 
 buildPythonApplication rec {
   pname = "rare";
-  version = "1.10.3";
+  version = "1.10.6";
 
   src = fetchFromGitHub {
     owner = "Dummerle";
     repo = "Rare";
     rev = "refs/tags/${version}";
-    hash = "sha256-7KER9gCpqjEKikQTVHsvwX6efCb9L0ut6OBjjLBW2tI=";
+    hash = "sha256-LtnWjrXlb1Ew+e+5EMymB1KB9WFP/uLnwb0OZb4xmrw=";
   };
 
   nativeBuildInputs = [
```

```suggestion
  nativeBuildInputs = [
    setuptools
```

Related to the `pyproject = true;` comment. It will pick the `setup.py` file to build it